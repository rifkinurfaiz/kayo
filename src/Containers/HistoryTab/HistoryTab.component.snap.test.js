import * as React from 'react';
import { shallow } from 'enzyme';

import HistoryTabComponent from './HistoryTab.component';

jest
  .mock('react', () => ({
    ...jest.requireActual('react'),
    useState: jest.fn()
  }))
  .mock('react-native-tab-view', () => ({
    SceneMap: jest.fn(),
    TabView: 'TabView',
    TabBar: 'TabBar'
  }))
  .mock('react-native-swiper', () => 'Swiper')
  .mock('./useHistoryTab', () =>
    jest.fn().mockReturnValue({
      index: 0,
      setIndex: jest.fn(),
      routes: [
        { key: 'first', title: 'Chart' },
        { key: 'second', title: 'Detail' }
      ]
    })
  );

describe('#HistoryTabComponent', () => {
  it('should render correctly', () => {
    expect(shallow(<HistoryTabComponent />)).toMatchSnapshot();
  });
});
