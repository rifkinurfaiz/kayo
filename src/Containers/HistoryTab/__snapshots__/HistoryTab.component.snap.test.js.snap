// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#HistoryTabComponent should render correctly 1`] = `
<TabView
  navigationState={
    Object {
      "index": 0,
      "routes": Array [
        Object {
          "key": "first",
          "title": "Chart",
        },
        Object {
          "key": "second",
          "title": "Detail",
        },
      ],
    }
  }
  onIndexChange={[MockFunction]}
  renderScene={[Function]}
  renderTabBar={[Function]}
  style={
    Object {
      "backgroundColor": "#E1E1E1",
    }
  }
  swipeEnabled={true}
/>
`;
